src/ralph/accounts/management/commands/ldap_sync.py

```diff
@@ -49,40 +52,55 @@ def __exit__(self, type, value, traceback):
         self.conn.unbind_s()
 
 
+@cache(seconds=600)
+def get_nested_groups():
+    """
+    Fetching users in nested group based on custom LDAP filter
+    (AUTH_LDAP_NESTED_FILTER) e.g. (memberOf:{}). AUTH_LDAP_NESTED_FILTER
+    is a simple dictonary where key is the name of group in DB, the value
+    contains DN for nested group.
+    """
+    # mapping from django group name to set of users (usernames) belonging to it
+    group_users = {}
+    # mapping from user (username) to set of groups DNs to which he belongs to
+    users_groups = defaultdict(set)
+    nested_groups = getattr(settings, 'AUTH_LDAP_NESTED_GROUPS', None)
+    if not nested_groups:
+        return
+    nested_filter = getattr(
+        settings, 'AUTH_LDAP_NESTED_FILTER', '(memberOf:{})'
+    )
+    logger.info('Fetching nested groups from LDAP')
+    with LDAPConnectionManager() as conn:
+        for ldap_group_name, ralph_group_name in nested_groups.items():
+            ldap_filter = nested_filter.format(ldap_group_name)
+            logger.info('Fetching {}'.format(ralph_group_name))
+            users = conn.search_s(
+                settings.AUTH_LDAP_USER_SEARCH_BASE,
+                ldap.SCOPE_SUBTREE,
+                '(&(objectClass={}){})'.format(
+                    settings.LDAP_SERVER_OBJECT_USER_CLASS, ldap_filter
+                )
+            )
+            logger.info('{} fetched'.format(ralph_group_name))
+            group_users[ralph_group_name] = set([
+                u[1][settings.AUTH_LDAP_USER_USERNAME_ATTR][0].decode('utf-8')  # noqa
```

Comment:
no, since `pyldap` (https://github.com/pyldap/pyldap) which we use underneath is using utf-8 everywhere (https://github.com/pyldap/pyldap/issues/1)